test/jdk/java/awt/Graphics/LCDTextAndGraphicsState.java

```diff
@@ -79,14 +107,81 @@ public void test3(Graphics g) {
     }
 
     public Dimension getPreferredSize() {
-        return new Dimension(500,600);
+        return new Dimension(500, 600);
     }
 
-    public static void main(String[] args) throws Exception {
+    private static void createInstructionUI() {
+        GridBagLayout layout = new GridBagLayout();
+        Panel mainControlPanel = new Panel(layout);
+        Panel resultButtonPanel = new Panel(layout);
+        GridBagConstraints gbc = new GridBagConstraints();
+        gbc.gridx = 0;
+        gbc.gridy = 0;
+        gbc.insets = new Insets(5, 15, 5, 15);
+        gbc.fill = GridBagConstraints.HORIZONTAL;
+
+        TextArea instructionTextArea = new TextArea();
+        instructionTextArea.setText(INSTRUCTIONS);
+        instructionTextArea.setEditable(false);
+        instructionTextArea.setBackground(Color.white);
+        mainControlPanel.add(instructionTextArea, gbc);
+
+        Button passButton = new Button("Pass");
+        passButton.setActionCommand("Pass");
+        passButton.addActionListener((ActionEvent e) -> {
+            testResult = true;
+            countDownLatch.countDown();
+        });
+
+        Button failButton = new Button("Fail");
+        failButton.setActionCommand("Fail");
```

Comment:
You don't use action command, so it's somewhat redundant.

Reply:
Removed actionCommand